py3status/modules/graphite.py

```diff
@@ -11,14 +11,11 @@
     format: you MUST use placeholders here to display data, see below.
         (default '')
     graphite_url: URL to your graphite server. (default '')
-    http_proxy: You can configure individual requests for the scheme HTTP,
-        see http://docs.python-requests.org/en/master/user/advanced/#proxies
-        (default: None)
-    https_proxy: You can configure individual requests for the scheme HTTPS,
-        see http://docs.python-requests.org/en/master/user/advanced/#proxies
-        (default: None)
     http_timeout: HTTP query timeout to graphite.
         (default 10)
+    proxy: You can configure the proxy with HTTP or HTTPS.
+        See: http://docs.python-requests.org/en/master/user/advanced/#proxies
```

Comment:
I'd remove this link.

1) It is specific to requests and the module should be neutral about the lib it happens to use.

2) The requests documentation is not very helpful.  I had to read the code to work out what the http/https was all about.

3) a simple example would be better for users to understand how to use